README.md

```diff
@@ -124,11 +124,11 @@ $ hoprd --help
 Options:
   --help                        Show help  [boolean]
   --version                     Show version number  [boolean]
-  --network                     Which network to run the HOPR node on  [choices: "ETHEREUM"] [default: "ETHEREUM"]
+  --environment                 Environment id which the node shall run on  [string] [choices: "hardhat-localhost", "hardhat-localhost2", "master-goerli", "debug-goerli", "tuttlingen", "prague"] [default: ""]
```

Comment:
no default ENV?

Reply:
Default environments are only set in releases.